### DISCUSSION TITLE
Interpolation mode=("bilinear","nearest")

### DISCUSSION THREAD

**Discussion by Gijz33** [Project-MONAI/MONAI#3327] [Q&A] [answered]
Hi!

I'm using monai.transforms.Spacingd to change the voxel size of my dataset. For interpolation of the data the option ("bilinear","nearest") is availabe. How does this interpolation work? Does it use a combination of bilinear and nearest interpolation?

I also tried to use only "bilinear" interpolation. When applying this to a binary label (for segmentation), it returns a label with multiple values (non-binary). How does Supervisedtrainer handle non-binary labels when binary labels are requested?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @Gijz33 ,

`("bilinear", "nearest")` means to apply different interpolation mode for the `keys` accordingly, for example here use `bilinear` for image and `nearest` for label.

Thanks.

**Reply by Gijz33** [reply]
Hi @Nic-Ma,
Thank you for your response. Your answer makes a lot of sense, don't know why I couldn't come up with it. 
Thanks again!

**Reply by Nic-Ma** [reply]
No problem, please feel free to raise a ticket if you face any other questions.

Thanks.
